classes/asserter/resolver.php

```diff
@@ -50,13 +52,28 @@ public function resolve($asserter)
 		{
 			$class = $this->checkClass($asserter);
 		}
-		else foreach ($this->namespaces as $namespace)
+		else
 		{
-			$class = $this->checkClass($namespace . '\\' . $asserter);
+			if (false === test::isValidIdentifier($asserter))
+			{
+				return null;
```

Comment:
Because, I don't know why, we can pass a `\\` as the asserter name. In this case, validation will fail. So in order to keep the same behavior… (same story ;-)).